libexec/estate-shrink/default.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#==============================================================================
+# Copyright (C) 2020-present Alces Flight Ltd.
+#
+# This file is part of Flight Action API.
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which is available at
+# <https://www.eclipse.org/legal/epl-2.0>, or alternative license
+# terms made available by Alces Flight Ltd - please direct inquiries
+# about licensing to licensing@alces-flight.com.
+#
+# Flight Action API is distributed in the hope that it will be useful, but
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR
+# IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS
+# OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY OR FITNESS FOR A
+# PARTICULAR PURPOSE. See the Eclipse Public License 2.0 for more
+# details.
+#
+# You should have received a copy of the Eclipse Public License 2.0
+# along with Flight Action API. If not, see:
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on Flight Action API, please visit:
+# https://github.com/openflighthpc/flight-action-api
+#===============================================================================
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Runs the estate-grow script with the 'Shrink' type
+__flight_ESTATE_action='Shrink'
+source "$DIR"/../estate-grow/default.sh
```

Comment:
For consistency with our other scripts this ought to be `"${SCRIPT_ROOT:-.}"/estate-grow/default.sh`.  There is then no need to set `DIR`.  Also, why is this using `source` instead of executing the script?

Reply:
Change made as requested

Reply:
https://github.com/openflighthpc/flight-action-api/pull/15/commits/aaef42cb16499fc7bd529fcabdc4e49819c43b7b